granulate_utils/metrics/yarn/utils.py

```diff
@@ -88,6 +96,18 @@ def is_first_resource_manager(self) -> bool:
     def first_resource_manager_webapp_address(self) -> str:
         return self.resource_manager_webapp_addresses[0]
 
+    def is_node_manager_running(self) -> bool:
```

Comment:
Unused

Reply:
Yeah for now, but have a look on @slicklash  [here](https://github.com/Granulate/granulate-utils/pull/217#discussion_r1420090556)